legit.py

```diff
@@ -0,0 +1,230 @@
+import logging
+from typing import Iterable
+
+from git import Repo
+from git.exc import GitCommandError
+
+log = logging.getLogger("legit")
+log.setLevel(logging.DEBUG)
+fh = logging.FileHandler("legit.log")
+fh.setLevel(logging.DEBUG)
+ch = logging.StreamHandler()
+ch.setLevel(logging.ERROR)
+formatter = logging.Formatter("%(asctime)s\t%(levelname)s -- %(processName)s %(filename)s:%(lineno)s -- %(message)s")
+ch.setFormatter(formatter)
+fh.setFormatter(formatter)
+log.addHandler(ch)
+log.addHandler(fh)
+
+
+class Repository(object):
+    """GitPython provides object model access to your git repository.
+    This class wraps up some fuctions of GitPython
+
+    Attributes:
+        repo : This is path to git repository. Most of the GitPython functions
+        use it to operate with repository.
+
+    """
+
+    def __init__(self, path: str) -> None:
+        """This command creates an empty Git repository - basically
```

Comment:
this is not a command, it is a method, namely a constructor. All constructors do the same thing - they initialize the object. This, as such, does not need to be explained, provided it's clear what the class is for.